xbmc/cores/RetroPlayer/guibridge/IGameCallback.h

```diff
@@ -25,6 +27,19 @@ class IGameCallback
    * \return The game client's ID, or empty if no game is being played
    */
   virtual std::string GameClientID() const = 0;
+
+  /*!
+   * \brief Get the game that is being played
+   *
+   * \return The path to the game, or empty if no game is being played
+   */
+  virtual std::string GetPlayingGame() const = 0;
+
+  virtual IPlayback* GetPlayback() const = 0;
```

Comment:
The class `IPlayback` exposes a lot of internal functionality. For example, an external client could call `Initialize()` multiple times and wreak havoc. Instead of exposing the entire interface, how about just exposing the `CreateSavestate()` and `LoadSavestate()` functions?